algosdk/v2client/algod.py

```diff
@@ -220,24 +262,29 @@ def block_info(
         """
         query = {"format": response_format}
         if block is None and round_num is None:
```

Comment:
Why can't this check \be in _specify_round_string?

Reply:
good catch!

Reply:
3e13b44dbe0b576a861363c0386e038ff7b18137